### DISCUSSION TITLE
How to make a TabPanel render even when it's not currently active

### DISCUSSION THREAD

**Discussion by crisshaker** [ariakit/ariakit#2174] [Questions] [answered]
I'm trying to re-create [this](https://twitter.com/_christian_o/status/1609234276242923521?s=20&t=ZDPh_t3CSZIeC90RxRBdKQ) using ariakit.

Basically, every TabPanel should render even when they are not active. The linked example uses headlessui by tailwind. Their TabPanel has a `static` prop that enables that behavior. I'm curious how I can achieve the same behavior using ariakit.

**Comment by diegohaz** [comment, marked as answer]
Try this:

```jsx
<TabPanel hidden={false}>
```

Also, an [example](https://github.com/ariakit/ariakit/blob/16a44a9ce68f5220999d8630673a0ec1456e6720/contributing.md#writing-another-example) is welcome.

**Reply by crisshaker** [reply]
Thanks a lot! That worked but.... How do I make it so that when I hit tab on the keyboard, it focuses the TabPanel of the current active Tab instead of selecting the very first TabPanel always? Thank you! I've attached a video to show the current behavior.


https://user-images.githubusercontent.com/36159205/210174932-c54e3d1e-ba38-4f07-a8b8-b895a4dcd177.mov

**Reply by crisshaker** [reply]
I found it. Using the disabled prop solved that. ariakit is soooo simple and powerful that it's easy to miss these simple things.

Can't wait to start contributing examples.

Happy New Year, @diegohaz

**Reply by crisshaker** [reply]
Here's the result btw.


https://user-images.githubusercontent.com/36159205/210175691-2492d566-b8f0-4256-a6f0-10a1155e645f.mov
